src/components/FloatingHelper/FloatingHelper.tsx

```diff
@@ -28,24 +38,43 @@ const FloatingHelperBO = withStylable<PopoverProps, {}>(
 );
 
 export const FloatingHelper: React.SFC<FloatingHelperProps> = props => {
-  const {children, content, ...rest} = props;
+  const {children, content, width, ...rest} = props;
+  const contentWidth = (typeof width) === 'number' ? `${width}px` : width;
+
   return (
     <FloatingHelperBO
       {...rest}
       showArrow
     >
       <Popover.Element>
           {children}
-        </Popover.Element>
-        <Popover.Content>
+      </Popover.Element>
+      <Popover.Content>
+        <div data-hook={DataHooks.contentWrapper} style={{width: contentWidth}}>
+          {props.withCloseButton && (
```

Comment:
Maybe we can just hide the closeButton with css?
Add a class and use `classNames` to control it's visibility.
It is just an approach, Not sure which is better.

Reply:
I would do that if it was needed for easy layout. But I use position `absolute` for the closeButton so it doesn't affect the layout.